src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -26,163 +26,155 @@
 
 namespace PrestaShopBundle\Controller\Admin\Sell\Catalog;
 
-use Exception;
-use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureConstraintException;
-use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureNotFoundException;
-use PrestaShop\PrestaShop\Core\Domain\Feature\Query\GetFeatureForEditing;
+use PrestaShopBundle\Bridge\AddActionInterface;
+use PrestaShopBundle\Bridge\AddActionTrait;
+use PrestaShopBundle\Bridge\Controller\ControllerBridgeInterface;
+use PrestaShopBundle\Bridge\Controller\ControllerConfiguration;
+use PrestaShopBundle\Bridge\Smarty\SmartyTrait;
 use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
 use PrestaShopBundle\Security\Annotation\AdminSecurity;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Response;
+use Tools;
 
 /**
  * Controller responsible for "Sell > Catalog > Attributes & Features > Features" page
  */
-class FeatureController extends FrameworkBundleAdminController
+class FeatureController extends FrameworkBundleAdminController implements ControllerBridgeInterface, AddActionInterface
```

Comment:
For readers: the result looks like https://github.com/PrestaShop/PrestaShop/blob/891c85e87c4aee1e2c8d810cf75aa4f6665bded3/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php